tests/main/validate-container-happy/test-snapd-validate-container-happy/meta/snap.yaml

```diff
@@ -0,0 +1,5 @@
+name: test-snapd-validate-container-happy
+version: 1.0
+apps:
+  validate-container:
+    command: bin/validate-container
```

Comment:
Hmm funny, looks like it's missing final `\n`, so if you were to `cat` the file in the terminal you'd end up with your prompt following the content of the last time.

Reply:
Fixed it, thanks :)